content/2015.07.whistler-use-cases-en.rst

```diff
@@ -0,0 +1,171 @@
+Summer Perspectives
+###################
+
+:lang: en
+:url: summer-perspectives
+:slug: post-whistler-usecases
+:date: 2015-07-07
+:summary: Work in progress and upcoming functionality over the coming months.
+
+.. note::
+
+    Translated from the French by `Dan Phrawzty <https://dark.ca/>`_.
+
+Mozilla has a tradition of "work weeks" whereby the members of our many distributed teams get together in a single physical location a few times per year; occasionally these work weeks involve the entire company, as was the case most recently in `Whistler, BC, Canada <http://www.openstreetmap.org/node/268148288#map=4/50.12/-122.95>`_.
+
+.. image:: {filename}/images/whistler-talks.jpg
+    :alt: "All Hands" talk about Lego, by @davidcrob - CC0
+    :align: center
+
+It was a great opportunity for our team to come together, to share our vision and ideas concerning data storage, and to collect use-cases for `Kinto <https://kinto.readthedocs.org>`_.
+
+Workshops and Self-Promotion
+============================
+
+Nicolas presented `Kinto.js <https://github.com/mozilla-services/kinto.js>`_ at a dedicated workshop in Whistler, the content of which now forms a `introductory tutorial <http://kintojs.readthedocs.org/en/latest/tutorial/>`_. The resulting application, simple though it appears, is a great way to learn the basic concepts behind synchronisation in Kinto. Furthermore, we didn't even need to set up a server - Rémy set one up that `resets itself every day <https://kinto.dev.mozaws.net/v1/>`_!
+
+As an aside, we chose to use Vanilla-JS for two reasons: firstly because it allowed us to avoid the framework wars, but also because it proves that we can do so much more with HTML5 and ES6 than was possible a few years ago.
+
+Furthermore, this small workshop helped us to realise that we still have large gaps in our documentation, particularly as regards the eco-system and the long-term, global vision for our family of projects. This is an area where we have resolved to do better.
+
+.. image:: {filename}/images/whistler-workshop.jpg
+    :alt: Kinto.js workshop - CC0
+    :align: center
+
+
+Mozilla Payments
+================
+
+As noted in a `previous blog post <http://www.servicedenuages.fr/la-gestion-des-permissions>`_ we developed a permissions model which allows Kinto to be much more useful with regards to payment and subscription systems.
+
```

Comment:
«allows Kinto to fulfill the requirements of payment and subscription tracking, our first use-case»
?
(*I wouldn't put systems in plural*)